extra_tests/not_impl_gen.py

```diff
@@ -17,6 +17,37 @@
 from pydoc import ModuleScanner
 
 
+DEAD_BATTERIES = {
```

Comment:
adding a comment about PEP594 here will be helpful in future to track the current state